### DISCUSSION TITLE
Only include a file if it exist

### DISCUSSION THREAD

**Discussion by freakalot** [wixtoolset/issues#6962] [Questions]
Hi, 

I need a way to only include a file if it exists, otherwise I define some variables. I tried to figure out how this could be done but I didn't get any solution now and the documentation doesn't really show up how something like this could be done. Maybe someone else did it before and can help me?

I guess it should look like this:
```
<?if fileExist?>
  <?include file?>
<?else?>
  <?define ...?>
<?endif?>
```

my question is: how can I find out if the file exists on the system where the solution is built?

Thanks
Frank

**Comment by robmen** [comment]
Check out the `Exists()` keyword in [MSBuild Conditions](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions). You'll build up from there.
